.github/workflows/maintenance-changelog.yml

```diff
@@ -3,6 +3,7 @@ name: cherry-pick changelog from release branch into default branch
 on:
   push:
     tags: ['*.*']
+    branches: ["*.x"]
```

Comment:
why is this change needed?

Reply:
Because otherwise this action tries to cherry-pick major releases (on main) into release branches (e.g. 15.x). However, we only want to allow it the other direction.